openshift/dynamic/newclient.py

```diff
@@ -0,0 +1,1009 @@
+import os
```

Comment:
Did you commit this intentionally?

Reply:
Heh, no, my bad. I think we have a better version of this patch somewhere, I need to talk to our guys, then I'll push it.